src/tests/components/EditProfile.test.js

```diff
@@ -0,0 +1,114 @@
+import { shallow } from "enzyme";
+import React from "react";
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import moxios from "moxios";
+import store from "../../store";
+
+import {
+  EditProfile,
+  mapStateToProps
+} from "../../components/userProfile/EditProfile";
+
+describe("editprofile component success tests", () => {
+  const initialState = {
+    profile: { isLoading: false, profile: null, message: null }
+  };
+  const props = {
+    profile: {
+      full_name: "",
+      bio: "",
+      image: ""
+    },
+
+    uploadImage: jest.fn(),
+    editProfileAction: jest.fn(),
+    history: { push: jest.fn() }
+  };
+  const nextProps = {
+    upDateSuccess: true
+  };
+  const mockFn = jest.fn();
+  const mockStore = configureStore([thunk]);
+  let store;
+  let wrapper = shallow(<EditProfile {...props} />);
+
+  beforeEach(() => {
+    store = mockStore(initialState);
+    moxios.install();
+  });
+
+  afterEach(function() {
+    moxios.uninstall();
+  });
+  it("should render without crushing", () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("should handle name change", () => {
+    const event = {
+      target: {
+        name: "full_name",
+        value: "emma ogwal"
+      }
+    };
+    wrapper.instance().handleNameChange(event);
+    expect(wrapper.instance().state.full_name).toBe("emma ogwal");
+  });
+  it("should handle bio change", () => {
+    const event = {
+      target: {
+        name: "bio",
+        value: "I am the fudge"
+      }
+    };
+    wrapper.instance().handleBioChange(event);
+    expect(wrapper.instance().state.bio).toBe("I am the fudge");
+  });
+  it("should handle image upload", () => {
+    const event = {
+      target: {
+        name: "image",
+        files: "files.jpg"
+      }
+    };
+    wrapper.instance().handleImageUpload(event);
+    expect(wrapper.instance().state.image).toBe("files.jpg");
+  });
+  it("should simulate image upload", () => {
```

Comment:
You have to expect or assert something in your test

Reply:
Done